osu.Game/Rulesets/UI/GameplaySampleTriggerSource.cs

```diff
@@ -26,16 +27,20 @@ public partial class GameplaySampleTriggerSource : CompositeDrawable
 
         private int nextHitSoundIndex;
 
-        private readonly Container<SkinnableSound> hitSounds;
+        protected readonly Container<SkinnableSound> HitSounds;
+        protected readonly DrawableAudioMixer AudioMixer;
 
         public GameplaySampleTriggerSource(HitObjectContainer hitObjectContainer)
         {
             this.hitObjectContainer = hitObjectContainer;
 
-            InternalChild = hitSounds = new Container<SkinnableSound>
+            InternalChild = AudioMixer = new DrawableAudioMixer
```

Comment:
This is the first usage of a nested `DrawableAudioMixer` which makes me very uneasy with it existing.

Can you explain why this was added? I can't find anything in PR or commit messages.

Reply:
Nested as in the `Drawable` hierarchy or as in nested mixers? Because I didn't think that was merged yet? This should just be another mixer at the same (top) level as the other (track/sfx) mixers?

It's used in `DrumSampleTriggerSource` to affect panning of the samples without having to set it per `HitObject`.

Reply:
Nested as in drawable yeah.

I think if you just want to change panning, you can do this using a `AudioContainer`. Should be a drop-in replacement.